include/deal.II/base/discrete_time.h

```diff
@@ -62,6 +62,18 @@ class DiscreteTime
   double
   get_current_time() const;
 
+  /**
+   * Return the next time that we reach if we advance time by one step.
+   */
+  double
+  get_next_time() const;
+
+  /**
+   * Return the previous time.
```

Comment:
Maybe say "Return the time we were at before `advance_time()` was called the last time" to make clear what "previous" corresponds to.

Reply:
That's a good way of saying it. I modified the doc.